pom.xml

```
@@ -94,12 +94,31 @@
           <configuration>
             <showWarnings>true</showWarnings>
             <showDeprecation>true</showDeprecation>
-            <failOnWarning>true</failOnWarning>
+            <fork>true</fork>
             <compilerArgs>
+              <arg>-XDcompilePolicy=simple</arg>
+              <arg>-Xplugin:ErrorProne</arg>
```

Should probably suppress warnings for classes generated by `proto` module:
```suggestion
              <arg>-Xplugin:ErrorProne -XepExcludedPaths:.*/generated-test-sources/protobuf/.*</arg>
```

Error Prone's `-XepDisableWarningsInGeneratedCode` does not seem to work because it relies on `@Generated` and the protobuf plugin does not add that yet, see https://github.com/xolstice/protobuf-maven-plugin/issues/29.